src/App.js

```diff
@@ -1,16 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messagesData, setMessages] = useState(chatMessages);
+
+  const likeChatButton = (updatedData) => {
+    const currentMessages = messagesData.map((messages) => {
+      if (messages.id === updatedData.id) {
+        return updatedData;
+      } else {
+        return messages;
+      }
+    });
+
+    setMessages(currentMessages);
+  };
```

Comment:
I might consider refactoring this, so that it doesn't update ALL of the message data, just the `liked` property